docs/features/backup_and_restore.md

```diff
@@ -6,7 +6,201 @@ hide_title: false
+(e.g. your RDS cluster) and re-run the database data migration
+module to migrate Collections, Providers, etc, from a (possibly recovered
+DynamoDB instance) then begin active migration of Files and Granules records.
+
+For larger datastores this approach may not work
+due to migration time/other limitations.  In that case, utilizing an RDS recovery
+approach in conjunction with the Dynamo to RDS reconciliation tooling
+may be appropriate.
+
+#### Backup and Restore with AWS RDS
+
+##### Configuring Database Backups
+
+For AWS RDS Aurora database deployments, AWS provides a host of database
+backup/integrity options, including [PITR (Point In Time
+Recovery)](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/USER_PIT.html)
+based on automated database backups and replay of transaction logs.
+
+For further information on RDS backup procedures, see the [AWS documentation](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/CHAP_CommonTasks.BackupRestore.html)
+
+##### Disaster Recovery
+
+To recover a Cumulus Postgres database in a disaster or data-loss scenario, you should perform the following steps:
+
+* If the Postgres database cluster exists/is still online, halt workflow
+  activity, then take the cluster offline/remove access.
+* If needed, recover the DynamoDB tables as noted in the
+  [DynamoDb](./backup_and_restore#dynamodb) section of this document.
+* Redeploy a new database cluster from your backup, matching as closely as possible to the DynamoDB
+  restore time.   See [AWS's
+  PIT recovery
+  instructions](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/USER_PIT.html)
+  and [DB Snapshot recovery
+  instructions](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/USER_RestoreFromSnapshot.html),
+  or the examples below for more information.
+* Configure your Cumulus deployment to utilize the new database cluster and re-deploy.
+* Run DynamoDB/RDS reconciliation tools and resolve any discrepancies.
+
+##### cumulus-rds-tf examples
+
+The following sections provide a walk through of a few recovery scenarios for the provided `cumulus-rds-tf`
+serverless module.
+
+***Point In Time Recovery***
+
+If you need recovery that exceeds the 1-day granularity of AWS's snapshots, you
+either must create and manually manage snapshots, or use Point In Time
+Recovery (PITR) if you still have the original cluster available.
+
+Unfortunately as terraform does not yet support RDS PITR (see:
+[github terraform-provider issue #5286](https://github.com/terraform-providers/terraform-provider-aws/issues/5286)),
+this requires a manual procedure.
+
+If you are using the `cumulus-rds-tf` module to deploy an RDS Aurora Serverless
+Postgres instance, the following procedure can be used to successfully spin up a duplicate
+cluster from backup in recovery scenarios where the database cluster is still viable:
+
+#### **1. Halt all ingest/remove access to the database to prevent Core processes from writing to the old cluster.**
```

Comment:
I don't really like the use of the slash here because it isn't clear whether it's an AND or an OR on halting ingest and/or removing access.

Reply:
7ae916cf8